cmd/juju/commands/main.go

```diff
@@ -155,15 +157,17 @@ func installProxy() error {
 	return nil
 }
 
-func (m main) maybeWarnJuju1x() (newInstall bool) {
+func (m main) maybeWarnJuju1x() (newInstall bool, jujuMsg string) {
+	jujuMsg = ""
```

Comment:
nit:
```suggestion
	var jujuMsg string
```

Reply:
Actually jujuMsg is already declared as a return var, so there is no need to re-declare it.